src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -152,6 +150,14 @@
              (when @show-delivery-state?
                [status/status outgoing-status])])]]]))))
 
+(defn on-long-press
+  [message-data context keyboard-shown]
+  (rf/dispatch [:dismiss-keyboard])
```

Comment:
Not for this PR, but what's the use of this event in the codebase? Isn't `rn/dismiss-keyboard!` sufficient?

Reply:
you can't use effects directly from view, event should be used instead

Reply:
I belive this is not an effect